loading-button-android/src/main/java/br/com/simplepass/loadingbutton/utils/Utils.kt

```diff
@@ -1,12 +1,7 @@
 package br.com.simplepass.loadingbutton.utils
 
 import android.content.Context
-import android.graphics.drawable.ColorDrawable
-import android.graphics.drawable.Drawable
-import android.graphics.drawable.GradientDrawable
-import android.graphics.drawable.InsetDrawable
-import android.graphics.drawable.RippleDrawable
-import android.graphics.drawable.StateListDrawable
+import android.graphics.drawable.*
```

Comment:
this project doesn't use wildcard import. Change only this and this PR is good to go.